### DISCUSSION TITLE
How to call Pyscript from a HA Script?

### DISCUSSION THREAD

**Discussion by dorbitbrown** [custom-components/pyscript#205] [Q&A]
Is there a way to invoke a Pyscript function from a HA Script?  

Conversely, is there a way to use Pyscript scripts or services as a HA script?  Specifically, I read that Alexa should be able to view HA Scripts, but in Alexa scenes I don't see any variables or services from Pyscript.

**Comment by stefanuytterhoeven** [comment]
create a service in pyscript. Such a service can be invoked by HA script. (select "call service" action, and select "pyscript.servicename"
